SocketRocket/SRWebSocket.m

```diff
@@ -1570,7 +1567,7 @@ - (NSOperationQueue *_Nullable)delegateOperationQueue
 
 @end
 
-//#define SR_ENABLE_LOG
+#define SR_ENABLE_LOG
```

Comment:
Hi guys. Great work and apologies in advance for coming late to the party. 

Did you mean to enable logging by default here for all builds?